test/test_filesystem_helper.cpp

```diff
@@ -286,6 +286,7 @@ TEST(TestFilesystemHelper, filesystem_manipulation)
   EXPECT_TRUE(rcpputils::fs::exists(file));
   EXPECT_TRUE(rcpputils::fs::is_regular_file(file));
   EXPECT_FALSE(rcpputils::fs::is_directory(file));
+  EXPECT_FALSE(rcpputils::fs::create_directories(file));
```

Comment:
I would also add empty case to make sure.
e.g) 
```
EXPECT_FALSE(rcpputils::fs::create_directories(""));
```

Reply:
Seems like a worthwhile addition to me.

Reply:
Great idea! a903aaa848b7fe09015d6e0c65fcd2a4bc9cf3b7